### DISCUSSION TITLE
Is it possible to provide static and dynamic control over the number of workers like php-fpm?

### DISCUSSION THREAD

**Discussion by ryan053857** [roadrunner-server/roadrunner#2068] [Q&A]
such as:

- max_worker_num
- start_worker_num
- idle_worker_num

and so on

**Comment by rustatian** [comment]
You can add workers on the fly: https://docs.roadrunner.dev/docs/php-worker/scaling, but no autoscaling atm.

**Reply by rustatian** [reply]
But I'm currently working on the dynamic allocator.
